src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -700,6 +826,7 @@ def run(self):
         telemetry.flush()
         while True:
             try:
+                self.completer.reset()
```

Comment:
Please add a comment to explain why we need to reset

Reply:
Done